src/Microsoft.Restier.Core/Conventions/ConventionBasedApiModelBuilder.cs

```diff
@@ -436,5 +340,131 @@ private void AddNavigationPropertyBindings(IEdmModel model)
                 }
             }
         }
+
+        internal class ModelMapper : IModelMapper
```

Comment:
CA can't get a knowledge that this class is actually instantiated, so an error is inevitable.
Also the InnerHandler member for not assigned.
This is kind of harassing, don't know if it could be circumvented.